src/main/java/HelloWorld.java

```diff
@@ -0,0 +1,5 @@
+public class HelloWorld {
+    public static void main(String[] args) {
+        System.out.println("Hello world!");
```

Comment:
Its ok this time, but next time be more creative:)

Reply:
Please change again!